src/www/cordova.webpack.js

```diff
@@ -0,0 +1,98 @@
+// Copyright 2020 The Outline Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const path = require('path');
+const CopyPlugin = require('copy-webpack-plugin');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const WebpackShellPluginNext = require('webpack-shell-plugin-next');
+const webpack = require('webpack');
+
+const OUTPUT_BASE = path.resolve(__dirname, '../../www');
+const GENERATE_CSS_RTL_LOADER = path.resolve(__dirname, '../../scripts/rtl_css_webpack.js');
+const BABEL_LOADER = {
+  loader: 'babel-loader',
+  options: {
+    presets: [
+      '@babel/preset-env',
+    ],
+  },
+};
+
+module.exports = {
+  entry: [
+    require.resolve('@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js'),
+    require.resolve('@babel/polyfill/dist/polyfill.min.js'),
+    require.resolve('web-animations-js/web-animations-next-lite.min.js'),
+    require.resolve('@webcomponents/webcomponentsjs/webcomponents-loader.js'),
+    path.resolve(__dirname, './style.css'),
+    path.resolve(__dirname, './app/cordova_main.ts'),
+  ],
+  mode: 'production',
+  target: 'web',
+  output: {path: OUTPUT_BASE, filename: 'main.js'},
+  module: {
+    rules: [
+      {
+        test: /\.ts(x)?$/,
+        exclude: /node_modules/,
+        use: [
+          BABEL_LOADER,
+          'ts-loader',
+          GENERATE_CSS_RTL_LOADER,
+        ],
+      },
+      {
+        test: /\.js$/,
+        exclude: /node_modules/,
```

Comment:
One think that is not clear to me is whether this exclusion prevents the transpilation of the node dependencies.

Reply:
That was a bug on my part. I updated the PR.

Essentially we want to transpile all the .js code (using Babel in this case) and run the RTL loader only on Outline code. Let me know what you think of the updated Webpack config.